app/views/users/show.html.erb

```diff
@@ -2,4 +2,36 @@
   <li><%= @users.name %></li>
   <li><%= @users.email %></li>
   <li><%= @users.created_at %></li>
-</ul>
\ No newline at end of file
+</ul>
+<div class='col-md-7'>
+    <h3><%= "#{@users.name.capitalize}'s" %> Friends</h3>
+    <ul>
+    <%@friendships = Friendship.where(user_id: @users.id, confirmed: true)%>
+      <% @friendships.each do |user| %>
+      <% x = User.find_by(id: user.friend_id)%>
+        <li>
+          <p><strong><%= "#{x.name}" %> </strong><%="#{x.email}" %> </p>
+          <p><%= link_to 'Profile', user_path(x.id) %></p>
+        </li>
+      <% end %>
+    </ul>
+  </div>
+<div class='col-md-5'>
+    <h3>Mutual Friends</h3>
+    <%@mutual_friendships = Friendship.where(user_id: current_user, confirmed: true, friend_id: (Friendship.where(user_id: @users.id, confirmed: true).select('friend_id'))) %>
```

Comment:
- [ ] Try to keep the queries out of the views, you can define this query in the Friendship model (same for all the other queries in your views.)

Reply:
We have removed the query from the view and placed it in the controller and fixed the error on the user page. We will work on the CSS after finishing with the Milestone 7